src/templates/installing/os/template.md

```diff
@@ -64,7 +64,7 @@ Switch to the built-in `admin` database:
 Create an administrative user (the is different from the `nodebb` user we'll create later). Replace the placeholder `<Enter a secure password>` with your own selected password. Be sure that the `<` and `>` are also not left behind.
 
 ```
-> db.createUser( { user: "admin", pwd: "<Enter a secure password>", roles: [ { role: "readWriteAnyDatabase", db: "admin" }, { role: "userAdminAnyDatabase", db: "admin" } ] } )
+> db.createUser( { user: "admin", pwd: "<Enter a secure password>", roles: [ { role: "root", db: "admin" } ] } )
```

Comment:
You need to be root to do some actions like delete a database. Since we give theses permissions to the admin and these enable `authorization`, there is no way to delete databases without having a root user. (from what I understood by having the issue while following the guide)

Reply:
This database user isn't used by NodeBB, so I don't think it really matters

Reply:
Oh I understand, that makes sense.